CodeGeneration/Sources/SyntaxSupport/Node.swift

```diff
@@ -106,10 +109,16 @@ public class Node {
           """
         experimentalSPI.with(\.trailingTrivia, .newline)
       }
+
       if forRaw {
         "@_spi(RawSyntax)"
       }
+
+      if isDeprecated, !ignoreAvailableAttribute {
+        #"@available(*, deprecated, message: "Will be removed in a future version.")"#
+      }
```

Comment:
A single place there is a enum that conforms to `CaseItaretable` and then we cannot apply this attribute.